### ISSUE TITLE
Is the project still alive?

### ISSUE THREAD

**Issue by hartmood** [krambox/moneymoney-ibkr#6]
After there has been no activity here for the last few months: Will the project be continued? If yes, great, if no, too bad.

**Comment by krambox** [comment]
Yes, the project is still active, but with low priority from my side.

**Comment by hartmood** [comment]
Thanks for letting me know. I would test and use it in any case
